src/Avalonia.Diagnostics/Avalonia.Diagnostics.csproj

```diff
@@ -14,6 +14,7 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\Avalonia.Controls.DataGrid\Avalonia.Controls.DataGrid.csproj" />
+    <ProjectReference Include="..\Avalonia.Dialogs\Avalonia.Dialogs.csproj" />
```

Comment:
Do we need this reference? In the code you have used SaveFileDialog which is available from controls package.
Dialogs package is not needed if you don't use managed file dialog if I remember correctly.